src/patternfly/components/ContextSelector/examples/context-selector-example.hbs

```diff
@@ -0,0 +1,145 @@
+{{#> context-selector id="context-selector-collapsed-example" context-selector--label-text="My Project"}}
```

Comment:
I don't think the label text and toggle text should match... I'm not sure how to handle this though. The label might read something like "Selected project"?

Reply:
going to go with that